### DISCUSSION TITLE
Can I convert the Graphiti based graphical editor to Sirius web ?

### DISCUSSION THREAD

**Discussion by lak-proddev** [eclipse-sirius/sirius-components#3114] [Q&A]
We have a Graphiti-based diagram editor used in our Studio. Can we move it to the Sirius Web-based editor?

**Comment by pcdavid** [comment]
You can reuse your EMF metamodel (ecore), but for the definition of the graphical part of the modeler you will need to rewrite it.

**Reply by lak-proddev** [reply]
Thank you very much for the quick reply.
